docker/complement/conf/start_for_complement.sh

```diff
@@ -44,27 +46,8 @@ esac
 
 
 if [[ -n "$SYNAPSE_COMPLEMENT_USE_WORKERS" ]]; then
-  # Specify the workers to test with
```

Comment:
The next day it occured to me that some use the Complement test suite with the CI. I'll handle this disappearing in a follow-up commit. But what's the best way? Ask the user to declare these values in their environment(or commandline) like they do now for `WORKERS` and `POSTGRES`? Seems messy, but would handle the missing values.

Reply:
For the record, I'm not a fan of duplication. But in the name of backwards compatibility, I suppose we have to make allowances. [Someone else agreed](https://github.com/matrix-org/synapse/blob/develop/pyproject.toml#L233).